Assets/Scripts/Player.cs

```diff
@@ -138,6 +149,16 @@ public void updatePowerUp()
         }
     }
 
+    /// <summary>
+    /// Play all player sounds theough here (as OneShot)
+    /// </summary>
+    /// <param name="clip"></param>
```

Comment:
use the param part of the xml commenting to describe the parameter of the function.  It helps with intellisense when calling the function somewhere else